### PR TITLE
Fix: show border for column resize

### PR DESCRIPTION
**Pull request by dan-mirescu**
Show colored border for column resize instead of a drag icon.

https://uipath.atlassian.net/browse/PLT-29993

### CODE REVIEW

**Review by gheorghitahurmuz — DISMISSED** [review]

**Review by anbalase — CHANGES_REQUESTED** [review]
with this change, the new UI communicates the resize anchor a bit too agressively. Pls sync w @alexandramehigh to get an idea if this is acceptable. I'd advise against. If we use this rendering, then the anchor should display/react near column header border and NOT on whole column header.

File: projects/angular/components/ui-grid/src/ui-grid.component.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@ ui-grid {
     $ui-grid-row-horizontal-padding: 24px;
     $ui-feature-cell-width: 50px;
     $ui-row-border-width: 1px;
-    $ui-header-resize-handle-width: 15px;
+    $ui-header-resize-handle-width: 1px;
```
Comment (anbalase) [inline projects/angular/components/ui-grid/src/ui-grid.component.scss:16]: are we making the handle only 1px? seems like a usability concern... pls also check w @alexandramehigh

Comment (anbalase) [reply projects/angular/components/ui-grid/src/ui-grid.component.scss:16]: 24px for optimum usability, though we could keep 15 for now if real-estate limitations won't allow it.

Comment (dan-mirescu) [reply projects/angular/components/ui-grid/src/ui-grid.component.scss:16]: The 1px was initially added to $ui-grid-default-spacing, resulting in the requested 6 pixels. However, the new changes contain improvements in this area.

**Comment by anbalase** [comment]
no changes pushed

**Comment by alexdudescu** [comment]
image
Having a seamless line from top to bottom when resizing (meaning row bottom borders won't show in between the blue elements) is complicated to pull off from a css standpoint and I think we should defer it. cc: @anbalase Wdyt?

**Comment by anbalase** [comment]
@alexdudescu I don't think we should add these changes to 14. These are just cosmetic changes

**Pull request closed without merging**